src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getDateFormatted, getPercentFormatted } from '../utils/helpers';
+
+class UserInfoSection extends React.Component {
+  render() {
+    const { userData } = this.props;
+
+    // Used to get today's date to show information current of
+    const today = new Date();
+    const todayFormatted = getDateFormatted(today);
+
+    // Check if percent is a number
+    let percentageBenefit = 'unavailable';
+    if (getPercentFormatted(userData.percentageBenefit)) {
+      percentageBenefit = getPercentFormatted(userData.percentageBenefit);
+    }
+
+    return (
+      <div className="gibstatus">
+        <h3 className="section-header">{userData.firstName} {userData.lastName}</h3>
+        <div className="usa-alert usa-alert-info">
+          <div className="usa-alert-body">
+            <h4 className="usa-alert-heading">This information is current as of {todayFormatted}</h4>
+          </div>
+        </div>
+        <div className="usa-grid-full section-line">
+          <div className="usa-width-one-third">
+            <span><strong>Date of Birth: </strong></span>
+          </div>
+          <div className="usa-width-one-third">
+            {getDateFormatted(userData.dateOfBirth)}
+          </div>
+        </div>
+        <div className="usa-grid-full section-line">
+          <div className="usa-width-one-third">
+            <span><strong>VA File Number: </strong></span>
+          </div>
+          <div className="usa-width-one-third">
+            {userData.vaFileNumber}
+          </div>
+        </div>
+        <div className="usa-grid-full section-line">
+          <div className="usa-width-one-third">
+            <span><strong>Regional Processing Office: </strong></span>
+          </div>
+          <div className="usa-width-one-third">
+            {userData.regionalProcessingOffice}
+          </div>
+        </div>
+        <div>
+          <h4>When You Can Receive Benefits</h4>
+          <div className="section-line">You are eligible to receive benefits between <strong>{getDateFormatted(userData.eligibilityDate)}</strong> and <strong>{getDateFormatted(userData.delimitingDate)}</strong></div>
```

Comment:
Maybe add a TODO here to find out how the enrollment data will indicate that the person is ineligible for Post-9/11 GIB benefits (so we can display the "Currently Disallowed" warning).